go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/iotaledger/iota-core
 go 1.20
 
 require (
+	github.com/AlecAivazis/survey/v2 v2.3.7
```

Comment:
make a separate go.mod for the evil-wallet and evil-spammer, please do not pollute the main go.mod